base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
WSL is a compatibility layer designed to make userspace look as close as possible to what it would be on Linux, because the userspace is all that applications care about. What do you mean by hybrid applications?

libuv doesn't call into the kernel on windows, it calls the win32 api. midipix is also userspace only, Microsoft won't be releasing public interfaces to the kernel any time soon. They released WSL, and we won't be getting anything dramatically different. WSL largely makes midipix obsolete.

If we're only using this for the sake of building code as opposed to runtime behavior switching, thrn we don't need to expose the kernel as a julia level variable. People want userspace level common names for the OS behavior.